### DISCUSSION TITLE
Highlight full combo maps in the map select

### DISCUSSION THREAD

**Discussion by serenityP** [ppy/osu#15271] [Ideas]
I am a catch player and achieve 4000+ S rank, but it is very exhausting for me to sort out S-rank maps that I did not full combo. Just want to get a better score.

This is my idea. 

![Full Combo UI](https://user-images.githubusercontent.com/13845063/138599876-864c9eed-c5f9-4a4b-a47e-107826fc67b5.jpg)

**Comment by bdach** [comment]
I'd rather imagine this as a filter that you can put in the search box rather than a visible element like that.

**Reply by serenityP** [reply]
Shall we add a filter called *Hide Full Combo Maps*?

**Reply by bdach** [reply]
I'd argue against this being some toggle or setting. The search box seems like a reasonable compromise between not adding to what is already a sensory overload of settings, while still facilitating relatively niche use cases (like yours invariably is).

**Reply by serenityP** [reply]
Input FC Maps=0 in the search box, is it workable?

**Reply by bdach** [reply]
That is what I'm thinking, yes. No promises whether it will happen or when, though. Would like more input from others whether this is something we want.

**Reply by serenityP** [reply]
Thanks. Some players who want clear 1 miss may need this.

**Comment by Gonzalo-Bruna** [comment]
Seems like something useful, probably the design and place could be improved by a good margin but the idea is good.
Probably showing somewhere at laederboards (for example when seeing your own scores or global scores) would be good to have as well.
